lasy/profiles/from_openpmd_profile.py

```diff
@@ -43,56 +45,176 @@ class FromOpenPMDProfile(FromArrayProfile):
         Prefix of the openPMD file from which the envelope is read.
         Only used when envelope=True.
         The provided iteration is read from <path>/<prefix>_%T.h5.
+
```

Comment:
There's one vertical space too much here

Reply:
Thanks! I fixed it.